packages/vaadin-combo-box/src/vaadin-combo-box-item.js

```diff
@@ -116,7 +116,10 @@ class ComboBoxItemElement extends ThemableMixin(DirMixin(PolymerElement)) {
     }
   }
 
-  _render() {
+  /**
+   * Runs all the renderers to possibly update the content.
+   */
+  runRenderers() {
```

Comment:
I wonder if it should be either public or private? 

Note, using either one or the other doesn't change anything so far `ComboBoxItemElement.runRenderers` is called only by `ComboBoxElement`.

Reply:
Indeed, doesn't matter that much as it's an internal module.